### PR TITLE
Created utility function for loading a class file

### PR DESCRIPTION
**Pull request by thk123**
To catch mistakes where the working directory is wrongly set verify that the class has been loaded. Also can reduce length of tests that load java bytecode substantially.

### CODE REVIEW

File: unit/src/java_bytecode/load_java_class.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#include <catch.hpp>
+#include <iostream>
+
+#include <util/config.h>
+#include <util/language.h>
+#include <util/suffix.h>
+
+#include <java_bytecode/java_bytecode_language.h>
+
+symbol_tablet load_java_class(
```
Comment (thk123) [inline unit/src/java_bytecode/load_java_class.cpp]: Document me!

**Comment by thk123** [comment]
Need to add the new file to the `Makefile` :man_facepalming:

File: unit/java_bytecode/java_bytecode_convert_class/convert_abstract_class.cpp [hunk 2]
```diff
@@ -125,14 +88,10 @@ SCENARIO("java_bytecode_convert_abstract_class",
...
-
-      symbol_tablet new_symbol_table;
-      java_lang->typecheck(new_symbol_table, "");
-
-      java_lang->final(new_symbol_table);
-
-      REQUIRE(new_symbol_table.has_symbol("java::Extendor"));
+      const symbol_tablet &new_symbol_table=
+        load_java_class(
+          "Extendor",
```
Comment (reuk) [inline unit/java_bytecode/java_bytecode_convert_class/convert_abstract_class.cpp]: There's a typo in this class name, but the class was added in a previous commit. Consider fixing this, but it's not too important (arguably out-of-scope and does not affect program behaviour)

File: unit/src/java_bytecode/load_java_class.h [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+/// Utility for loading and parsing a specified java class file, returning
+/// the symbol table generated by this.
+
+#ifndef CPROVER_SRC_JAVA_BYTECODE_LOAD_JAVA_CLASS_H
+#define CPROVER_SRC_JAVA_BYTECODE_LOAD_JAVA_CLASS_H
+
+#include <util/symbol_table.h>
+
+symbol_tablet load_java_class(
+  const std::string java_class_name,
```
Comment (reuk) [inline unit/src/java_bytecode/load_java_class.h]: Should be a const ref

**Review by reuk — APPROVED** [review]
Looks good to me, just a couple of really minor issues to check.

**Review by smowton — APPROVED** [review]

File: unit/java_bytecode/java_bytecode_convert_class/convert_abstract_class.cpp (hunk 2 above)
Comment (thk123) [reply unit/java_bytecode/java_bytecode_convert_class/convert_abstract_class.cpp]: Updated to use `Extender` which is I think what you meant?

**Pull request merged**